src/classes/UTIL_UnitTestData_TEST.cls

```diff
@@ -451,7 +451,7 @@ public class UTIL_UnitTestData_TEST {
     public static Course_Offering__c createCourseOffering(Id courseId, Id termId) {
         Account acct;
         if (courseId == null || termId == null) {
-            acct = UTIL_UnitTestData_API.getMultipleTestAccounts(1, UTIL_Describe_API.getAdminAccRecTypeID())[0];
+            acct = UTIL_UnitTestData_API.getMultipleTestAccounts(1, UTIL_Describe_API.getAcademicAccRecTypeID())[0];
```

Comment:
@robertbwatson does this have the possibility of being null/empty if the customer org has deleted the Academic record type? I think our UTIL describe stuff takes care of this for us but thought I'd ask.

Reply:
Hmm - it's possible that null could be returned - but I think in customer orgs, we can't ever guarantee that unit tests will always pass if they make modifications to their configuration. Technically the same could be said if the customer deleted/disabled the administrative record type as well.